browser/brave_content_browser_client.cc

```diff
@@ -76,39 +78,24 @@ using extensions::ChromeContentBrowserClientExtensionsPart;
 #include "brave/browser/tor/tor_profile_service_factory.h"
 #endif
 
-namespace {
-
-bool HandleURLOverrideRewrite(GURL* url,
-                              content::BrowserContext* browser_context) {
-  // redirect sync-internals
-  if (url->host() == chrome::kChromeUISyncInternalsHost ||
-      url->host() == chrome::kChromeUISyncHost) {
-    GURL::Replacements replacements;
-    replacements.SetHostStr(chrome::kChromeUISyncHost);
-    *url = url->ReplaceComponents(replacements);
-    return true;
-  }
-
-  // no special win10 welcome page
-  if (url->host() == chrome::kChromeUIWelcomeWin10Host ||
-      url->host() == chrome::kChromeUIWelcomeHost) {
-    *url = GURL(chrome::kChromeUIWelcomeURL);
-    return true;
-  }
+#if BUILDFLAG(BRAVE_WALLET_ENABLED)
+#include "brave/browser/extensions/brave_component_loader.h"
```

Comment:
this stuff also needs to be in an `ENABLE_EXTENSIONS` guard

Reply:
will be handled instead via `brave_wallet_enabled = !is_official_build && enable_extensions` in the `gni` file